package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Iteration over cursor-like objects with async/await",
   "main": "promise-iterate.js",
   "scripts": {
-    "test": "mocha"
+    "build": "rm -rf dist/node && babel src -d dist/node",
+    "prepublish": "yarn run build",
```

Comment:
make this prepublishOnly please so it won't run in Travis/when you check the module out

Reply:
(I think that might be a deficit in the migration script)